src/components/ui/AspectRatio/aspect-ratio.scss

```diff
@@ -0,0 +1,109 @@
+[data-store-aspect-ratio-wrapper] {
+  position: relative;
+  display: block;
+  width: 100%;
+  height: 0;
+  overflow: hidden;
+
+  &[data-variant="1"] {
```

Comment:
Do you think it's better if we split `Ratio` and `InverseRatio` values here. Or it's okay to keep them listed all together?  
(Ex: adding a comment to separate them) If it's not needed I'm also fine with this list :)